src/sources/prometheus/scrape.rs

```diff
@@ -255,7 +259,7 @@ fn prometheus(
 
                                 match parser::parse_text(&body) {
                                     Ok(events) => {
-                                        emit!(&PrometheusEventReceived {
+                                        emit!(&PrometheusEventsReceived {
                                             byte_size,
```

Comment:
I don't think this `byte_size` is right. It is the same as what is being used for `component_received_bytes_total`. Instead, it should be summing up `ByteSizeOf::size_of` the events.